daemon/graphdriver/driver_solaris.go

```diff
@@ -0,0 +1,63 @@
+package graphdriver
```

Comment:
Should this have the build tag `// +build solaris,cgo` too, like `daemon_solaris.go` ?